internal/webhook/v1/alertmanager_config_secret_webhook.go

```diff
@@ -148,7 +148,7 @@ func (v *AlertmanagerConfigSecretCustomValidator) validateTenant(ctx context.Con
 func validateAlertmanagerConfig(ctx context.Context, secret *corev1.Secret) error {
 	_, err := alertmanager.ExtractAlertmanagerConfig(ctx, secret)
 	if err != nil {
-		return err
+		return fmt.Errorf("alertmanager configuration validation failed: %w. Note: If you're using a newer Alertmanager feature, it might not be supported yet by the Grafana fork (grafana/prometheus-alertmanager) used by Mimir. Check the Mimir documentation for supported configuration options", err)
```

Comment:
Do we know where is this Mimir documentation ? I could not find it.

Reply:
Technically, there's none. The only place to check is the commit in the grafana fork